### DISCUSSION TITLE
Are spaces _really_ ignored in a prompt? That's not my experience...

### DISCUSSION THREAD

**Discussion by andrewconnell** [microsoft/teams-sdk#1346] [Q&A]
The docs state that [spaces in prompt references are ignored](https://github.com/microsoft/teams-ai/blob/main/getting-started/CONCEPTS/PROMPTS.md?plain=1#L61), but that's not been my experience... 

I'm getting errors when I reference functions  (`{{ getLightStatus }}` doesn't work, but `{{getLightStatus}}` does) or something nested (`{{ $conversation.lists }}` doesn't work, but `{{$conversation.lists}}` does).

Are the docs correct, is there a bug, or are there nuances that should be called out in the docs?

**Comment by corinagum** [comment]
Hi @andrewconnell, thanks for filing the discussion item. 

We did have a bug filed that has since been fixed: https://github.com/microsoft/teams-ai/pull/1206. However, we haven't done a release yet that includes these bug fixes. I am currently discussing with our team the plan to do a release and we'll get back to you on that shortly!

**Reply by corinagum** [reply]
@andrewconnell Thanks for your patience! Our plan is to do a release early next week. Please stay tuned!

**Reply by corinagum** [reply]
@andrewconnell Thanks again for waiting! We have done a [new release](https://github.com/microsoft/teams-ai/releases/tag/js-1.1.1) that includes the fix for the template whitespace bug. Once I see confirmation that you see this message, I will close this discussion unless you have a follow-up question.

**Reply by andrewconnell** [reply]
Thanks for the follow-up @corinagum ... I'll close it out.
